docs/using-the-jdbc-driver/UsingTheJdbcDriver.md

```diff
@@ -66,14 +66,22 @@ Plugins are loaded and managed through the Connection Plugin Manager and may be
 | `wrapperProfileName` | `String` | No       | Driver configuration profile name. Instead of listing plugin codes with `wrapperPlugins`, the driver profile can be set with this parameter. <br><br> Example: See [below](#configuration-profiles). | `null`        |
 
 To use a built-in plugin, specify its relevant plugin code for the `wrapperPlugins`.
```

Comment:
I like the name `defaultPlugins` instead of `wrapperPlugins`

Reply:
I am not sure if defaultPlugins makes sense, it is a parameter for setting the wrapper plugins to be used.